carla-spice-sync-service/SpiceUtils.cs

```diff
@@ -1433,35 +1402,35 @@ public async Task SendFoundWorkers(PerformContext hangfireContext)
                     else
                     {
                         hangfireContext.WriteLine($"Failed to send worker {screeningRequest.RecordIdentifier} to SPD");
-                        _logger.LogError($"Failed to send worker {screeningRequest.RecordIdentifier} to SPD");
+                        Log.Logger.Error($"Failed to send worker {screeningRequest.RecordIdentifier} to SPD");
                     }
                 }
             }
 
-            _logger.LogError("End of SendFoundWorkers Job");
+            Log.Logger.Error("End of SendFoundWorkers Job");
             hangfireContext.WriteLine("End of SendFoundWorkers Job");
         }
 
         [DisableConcurrentExecution(timeoutInSeconds: 10 * 60)]
         public async Task SendFoundApplications(PerformContext hangfireContext)
         {
             string[] select = { "adoxio_applicationtypeid" };
-            IList<MicrosoftDynamicsCRMadoxioApplicationtype> selectedAppTypes = _dynamicsClient.Applicationtypes.Get(filter: "adoxio_requiressecurityscreening eq true", select: select).Value;
+            IList<MicrosoftDynamicsCRMadoxioApplicationtype> selectedAppTypes = _dynamicsClient.Applicationtypes.Get(filter: "adoxio_haslesection eq true", select: select).Value;
```

Comment:
I guess the field got changed in Dynamics? It used to be `adoxio_requiressecurityscreening`. Not sure how it relates to `haslesection`...

Reply:
Yes, unfortunately I didn't realize this flag was being used by the service; we use it to toggle some items in the front end and part of the migration to the new flow involved disabling the adoxio_requiressecurityscreening flag.  adoxio_haslesection, when combined with the filter to look at the Sent to SPD flag, will have the correct result though.